README.md

```diff
@@ -52,3 +52,9 @@ We love contributions! Check out the [Contribution guide](https://github.com/Kri
 ### Area#3: Natural Language Processing 
     - UseCase#1 : Credit Card Fraud Detection
     - UseCase#2 : Spam Or Ham classification 
+
+### Area#4: RoshanML-AI
+
+Roshan Lamichhane's Machine Learning Projects 
+
```

Comment:
As a practice in ML-Ai community, we will have authors details shown in the project directory and specially in README.md file of that work.

Master file will show the contributors here for visibility. 

https://github.com/roshancode/ml-ai#contributing